src/main/java/dinkplugin/util/QuestUtils.java

```diff
@@ -46,21 +53,20 @@ public class QuestUtils {
     private final Collection<String> RFD_TAGS = ImmutableList.of("Another Cook", "freed", "defeated", "saved");
     private final Collection<String> WORD_QUEST_IN_NAME_TAGS = ImmutableList.of("Another Cook", "Doric", "Heroes", "Legends", "Observatory", "Olaf", "Waterfall");
 
+    @Nullable
     public String parseQuestWidget(final String text) {
-        // "You have completed The Corsair Curse!"
-        final Matcher questMatch1 = QUEST_PATTERN_1.matcher(text);
-        // "'One Small Favour' completed!"
-        final Matcher questMatch2 = QUEST_PATTERN_2.matcher(text);
-        final Matcher questMatchFinal = questMatch1.matches() ? questMatch1 : questMatch2;
-        if (!questMatchFinal.matches()) {
-            return "Unable to find quest name!";
+        Matcher matcher = getMatcher(text);
+        if (matcher == null) {
+            log.warn("Unable to match quest: {}", text);
+            return null;
```

Comment:
We could return `"Unknown quest"` or `text` here if we want a notification fired for quest completions that don't match the regexps... thoughts?

Reply:
`null` seems fine here imo until we feel like we have a use for the distinction